From 9b61ec7c58e8106c28d01b33ad0b263aefb1f347 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ji=C5=99=C3=AD=20Klime=C5=A1?= <jklimes@redhat.com>
Date: Tue, 1 Feb 2011 16:47:04 +0100
Subject: [PATCH] libnm-util: make NM_SETTING_IP4_CONFIG_DHCP_SEND_HOSTNAME
 property TRUE by default (rh #488975)
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

That lets NM send a hostname to DHCP server without requiring the user
editing the parameter manually (there no GUI for this either).

Signed-off-by: Jiří Klimeš <jklimes@redhat.com>
---
 libnm-util/nm-setting-ip4-config.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/libnm-util/nm-setting-ip4-config.c b/libnm-util/nm-setting-ip4-config.c
index 186e41e..ec8de7c 100644
--- a/libnm-util/nm-setting-ip4-config.c
+++ b/libnm-util/nm-setting-ip4-config.c
@@ -950,7 +950,7 @@ nm_setting_ip4_config_class_init (NMSettingIP4ConfigClass *setting_class)
 						   "the 'dhcp-hostname' property is empty and this "
 						   "property is TRUE, the current persistent hostname "
 						   "of the computer is sent.",
-						   FALSE,
+						   TRUE,
 						   G_PARAM_READWRITE | G_PARAM_CONSTRUCT | NM_SETTING_PARAM_SERIALIZE));
 
 	/**
-- 
1.7.6

